__tests__/app.test.js

```diff
@@ -66,19 +66,36 @@ describe("app", () => {
             });
           });
       });
-      test("returns in date descending order, this shows the oldest and newest film", () => {
+      test("returns in date descending order", () => {
         return request(app)
           .get("/api/reviews")
           .expect(200)
           .then((res) => {
-            expect(res.body.reviews[0].created_at).toBe(
-              "2021-01-25T11:16:54.963Z"
-            );
-            expect(
-              res.body.reviews[res.body.reviews.length - 1].created_at
-            ).toBe("1970-01-10T02:08:38.400Z");
+            expect(res.body.reviews).toBeSorted("created_at", {
+              descending: true,
+            });
           });
       });
+      describe("GET /api/reviews", () => {
+        test("returns an object with the correct review_id from the query query with an owner, title, review_id, category, review_img_url, created_at, votes, designer", () => {
+          return request(app)
+            .get("/api/reviews/1")
+            .expect(200)
+            .then((res) => {
+              const review = res.body.review;
+              expect(typeof review).toBe("object");
+              expect(review.hasOwnProperty("owner")).toBe(true);
+              expect(review.hasOwnProperty("title")).toBe(true);
+              expect(review.hasOwnProperty("review_id")).toBe(true);
+              expect(review.hasOwnProperty("category")).toBe(true);
+              expect(review.hasOwnProperty("review_img_url")).toBe(true);
+              expect(review.hasOwnProperty("created_at")).toBe(true);
+              expect(review.hasOwnProperty("votes")).toBe(true);
+              expect(review.hasOwnProperty("designer")).toBe(true);
+              expect(review.hasOwnProperty("review_body")).toBe(true);
+            });
+        });
+      });
     });
   });
 });
```

Comment:
Where are your error handling tests? You should do one for a 400 and one for a 404.